docstore/drivertest/drivertest.go

```diff
@@ -183,67 +193,54 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker, ct CodecTester,
 				t.Fatalf("AsTest.Name is required")
 			}
 			t.Run(st.Name(), func(t *testing.T) {
-				withTwoKeyCollection(t, newHarness, func(t *testing.T, coll *docstore.Collection) {
+				withCollection(t, newHarness, TwoKey, func(t *testing.T, _ Harness, coll *docstore.Collection) {
 					testAs(t, coll, st)
 				})
 			})
 		}
 	})
 }
 
-func withHarnessAndCollection(t *testing.T, newHarness HarnessMaker, f func(*testing.T, context.Context, Harness, *ds.Collection)) {
+func withCollection(t *testing.T, newHarness HarnessMaker, kind CollectionKind, f func(*testing.T, Harness, *ds.Collection)) {
 	ctx := context.Background()
 	h, err := newHarness(ctx, t)
 	if err != nil {
 		t.Fatal(err)
 	}
 	defer h.Close()
 
-	dc, err := h.MakeCollection(ctx)
+	withColl(t, h, kind, f)
+}
+
+func withColl(t *testing.T, h Harness, kind CollectionKind, f func(*testing.T, Harness, *ds.Collection)) {
```

Comment:
nit: move this above or below the other with...Collection methods and maybe add a comment to it.

Reply:
Done.

I agree about the separate PR. You are quite right.